### PR TITLE
Adds a pressure control system to cryotubes

### LINKED ISSUE #66151: New Cryo Sleepmix Bug
## Reproduction:
Step 1: Upgrade the freezer connected to any of the new cryogenic systems.
Step 2: Cool the gas to extremely low temperatures.
Step 3: Realize the gas shrinks down so much that it no longer provides sleep, eliminating the "cooler = better" aspect of cryoxadone.
Step 4: Accidentally kill your patient because they're just getting burned by the cold temperatures and not healed.
Step 5: Realize cryo is kind of shit now.

Personal Opinion:
I like the idea of gasses being used, but also, internals should bypass it (If they don't already, plasmamen don't breathe oxygen.) and also adding a pressure option to the cryotubes to forcefully change the amount of pressure regardless of the moles/pressure/temperature in the tubing, kind of like an internal pump of sorts. (This could even be upgraded with a better matter bin to allow for higher pressures to be set like with temperature on freezers when upgrading their lasers.)

### PR DESCRIPTION
**Pull request by RikuTheKiller**
## About The Pull Request

fixes #66151

Cryoxadone is made so that the lower the temperature of it's user is, the more effective it is.

Since gas shrinks whenever it's cooled down and cryotubes make their occupants unconscious via an anesthetic mix, cooling the gas down using an upgraded freezer causes an issue where the pressure of the gas is too low for it to do anything. (Cryoxadone requires the user to be unconscious in order to work.)

This PR adds a pressure control system to cryotubes that defaults to 25 kPa. (Results in a partial pressure of 16 or so for oxygen.)
## Why It's Good For The Game

Upgraded cryotube setups not being deathtraps prevents dumb accidents. (And also makes sense.)
## Changelog
:cl:
add: Cryotubes now have a pressure control system.
/:cl:

### CODE REVIEW

**Comment by Overuseddart** [comment]
Me after making the cryo tube 0 KPA

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 1]
```diff
@@ -338,12 +347,12 @@
 		update_parents()
 
 /obj/machinery/atmospherics/components/unary/cryo_cell/handle_internal_lifeform(mob/lifeform_inside_me, breath_request)
-
+	var/datum/gas_mixture/air1 = airs[1]
```
Comment (Time-Green) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:350]: Code looks good! Although could you change the name of this? I have no idea what air1] is

**Review by Time-Green — APPROVED** [review]

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm [hunk 2]
```diff
@@ -452,6 +461,13 @@
 		ui = new(user, src, "Cryo", name)
 		ui.open()
 
+/obj/machinery/atmospherics/components/unary/cryo_cell/ui_static_data()
+	. = list (
+		"defaultReleasePressure" = CRYO_DEFAULT_RELEASE_PRESSURE,
+		"minReleasePressure" = CRYO_MIN_RELEASE_PRESSURE,
+		"maxReleasePressure" = CRYO_MAX_RELEASE_PRESSURE,
+	)
```
Comment (JohnFulpWillard) [inline code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:469]:
```suggestion
	var/list/data = list()
	data["defaultReleasePressure"] = CRYO_DEFAULT_RELEASE_PRESSURE,
	data["minReleasePressure"] = CRYO_MIN_RELEASE_PRESSURE,
	data["maxReleasePressure"] = CRYO_MAX_RELEASE_PRESSURE,
	return data
```

Comment (RikuTheKiller) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:469]: Sure? (copied from tank code, but okay)

File: code/modules/atmospherics/machinery/components/unary_devices/cryo.dm (hunk 1 above)
Comment (RikuTheKiller) [reply code/modules/atmospherics/machinery/components/unary_devices/cryo.dm:350]: It's confusing but also what do you really name it, it's the air of the first pipe node. (The cryotube's pipe node.) Also, it's named that everywhere else in the cryotube code so... there's that.

**Comment by Ghilker** [comment]
> Is there a way to achieve this effect in game with atmospherics equipment?
> 
> I'd rather people build this functionality as part of upgrading the mix rather than just have it given to them for free.

Yes, you just need a pressure valve and a pressure pump

**Comment by optimumtact** [comment]
I'm against it this change then

**Comment by vincentiusvin** [comment]
> Yes, you just need a pressure valve and a pressure pump

I'm not sure that's possible. Can you elaborate?

**Comment by RikuTheKiller** [comment]
> I'm against it this change then

There is no reasonable way to do it with atmos equipment. At best you can compress the gas, but cold gas is STILL too not-dense for that unless you want to add an unhealthy amount of gas in which case, yes, it's theoretically possible. But no-one is going to do it.

**Comment by RikuTheKiller** [comment]
I'd prefer if this was reopened, even if it's 100% going to have merge conflicts.

**Pull request closed without merging**